src/write.c

```diff
@@ -2040,7 +2414,7 @@ avifResult avifEncoderFinish(avifEncoder * encoder, avifRWData * output)
         // Use multiple passes to pack in the following order:
         //   * Pass 0: metadata (Exif/XMP)
         //   * Pass 1: alpha (AV1)
-        //   * Pass 2: all other item data (AV1 color)
+        //   * Pass 2: all other item data (AV1 color, gain map)
```

Comment:
Should gain maps come before color for the same reason as alpha?

Unrelated note: These are not really "passes" but just data ordering. Some refactoring could make it clearer.